compiler/passes/createTaskFunctions.cpp

```diff
@@ -575,17 +575,21 @@ addVarsToFormalsActuals(FnSymbol* fn, SymbolMap& vars,
                         CallExpr* call, bool isCoforall)
 {
   Expr *redRef1 = NULL, *redRef2 = NULL;
-  form_Map(SymbolMapElem, e, vars) {
-      Symbol* sym = e->key;
-      if (e->value != markPruned) {
+
+  SymbolMapVector elts = sortedSymbolMapElts(vars);
+  for (auto pair: elts) {
```

Comment:
I believe it is possible and the implied `elts` would survive until the end of the loop. I did it this way because I thought it was clearer but I don't have a strong preference here. In any case it is a style improvement so I'd like to leave that for a future change if we decide to do it.